core/controlplane/config/templates/cloud-config-controller

```diff
@@ -690,6 +690,8 @@ write_files:
         selector:
           matchLabels:
             k8s-app: calico-node
+        updateStrategy:
+          type: RollingUpdate
```

Comment:
@c-knowles Thanks for chiming in! Yes, this will break the 1.5.x support.

I have never intended to keep supporting k8s 1.5.x myself. In that regard, this change is OK for me. However, I know some people still likes to go with 1.5.x.

I wish someone helps me bringing back the 1.5.x support but if no one is going to, I'll do, probably after listing up all the changes required to re-add 1.5.x support  😃